backend/lib/endpoints/schema/users.js

```diff
@@ -4,6 +4,18 @@ const getUserByIdSchema = {
   params: S.object().prop("userId", S.string().required()),
 };
 
+const createProfileSchema = {
+  body: S.object()
+    .prop("firstName", S.string().required())
```

Comment:
I would add `.additionalProperties(false)` so that we don't allow any additional properties that are not defined in this schema.

Reply:
added